src/normalize.css

```diff
@@ -0,0 +1,350 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
+
+/* Document
+   ========================================================================== */
+
+/**
+ * 1. Correct the line height in all browsers.
+ * 2. Prevent adjustments of font size after orientation changes in iOS.
+ */
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)

Reply:
removed normalize.css